### ISSUE TITLE
Will it work while nobody is on the site?

### ISSUE THREAD

**Issue by babilunga** [shawnscoding/reactjs-crontab#11]
Hello.

I developed a simple React App and hosted it on Heroku.
I need this my app to send some stuff at 00:00 UTC and there is a huge chance that nobody will be on the site at this particular time. Will it work in this case?

thank you for the attention.

**Comment by maxhash** [comment]
No, it only works inside the browser, so if the browser is not the cron is running.
